lib/ui/screen-components/protected/restaurant/dashboard/restaurant-stats-table/index.tsx

```
@@ -17,70 +21,87 @@ import { DASHBOARD_PAYMENT_METHOD } from '@/lib/utils/constants';
+
+  return (
+    <div className="p-3 bg-gray-50 rounded-lg shadow-md space-y-6">
+      {paymentMethod.map((method: string) => {
+        const key: keyof typeof dashboardOrderSalesDetailsByPaymentMethod =
+          method as keyof typeof dashboardOrderSalesDetailsByPaymentMethod;
+
+        if (key !== 'all' && key !== 'cod' && key !== 'card') return;
+
+        return (
+          <>
+            <div className="flex flex-col space-y-2">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)